pattern_test.go

```diff
@@ -37,3 +38,17 @@ func ExampleFuture() {
 	// Ret: foobar
 	// Err: <nil>
 }
+
+func BenchmarkCombineFuncs(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		u.CombineFuncs(func() {}, func() { fmt.Println("A") }, func() { fmt.Println("B") })
```

Comment:
not sure if it will change or not, but I suggest moving the two funcs initialization at the top of the benchmark function and use a b.ResetTimer

keep in mind that I want to bench my code, not the code around